flow-nodes/api-builder-plugin-md5/test/test.js

```diff
@@ -1,66 +1,57 @@
 const getPlugin = require('../src');
 const actions = require('../src/actions');
 const { expect } = require('chai');
-const { MockRuntime } = require('@axway/api-builder-sdk');
+const { MockRuntime } = require('@axway/api-builder-test-utils');
 
 describe('flow-node md5', () => {
-	let runtime;
-	before(async () => runtime = new MockRuntime(await getPlugin()));
+	let plugin;
+	let flowNode;
+	before(async () => {
+		plugin = await MockRuntime.loadPlugin(getPlugin);
+		flowNode = plugin.getFlowNode('md5');
+	});
 
 	describe('#constructor', () => {
 		it('should define flow-nodes', () => {
 			expect(actions).to.be.an('object');
 			expect(actions.digest).to.be.a('function');
```

Comment:
the SDK already ensures that method "digest" has action function "digest", so technically that doesn't need to be tested.  however, it does catch if "digest" was deleted from the definition and from actions, but forgotten in tests.  If this check was removed, then `await flowNode.digest` would fail, perhaps less clean?

Reply:
There's flownode.getMethods() for this purpose.

Reply:
Let's leave this as it is here - if we want to make this change, let's do it in the template.